AWS Publish Templates.yml

```diff
@@ -339,6 +347,10 @@ stages:
           targetFolder: '$(FolderNameProduction)'
           filesAcl: 'public-read'
 
+      - pwsh: |
+          Get-ChildItem '$(Pipeline.Workspace)/_lansa_aws-templates-mp/CloudFormationWindows'
+          Get-Content '$(Pipeline.Workspace)/_lansa_aws-templates-mp/CloudFormationWindows/lansa-stack-type-win.cfn.template'
+        
```

Comment:
Please explain this change

Reply:
Both lines need to be commented/delete. It was added just to check whether s3 getting correct updated file or not.